test/dataframerow.jl

```diff
@@ -121,6 +124,7 @@ end
     df = deepcopy(ref_df)
     df2 = DataFrame(a = [1, 2, 3])
 
+    @test !isequal(DataFrameRow(df, 1, :), DataFrameRow(df2, 1, :))
```

Comment:
Maybe also test a case where it's true (including with a missing value)?

Reply:
added